generator/internal/genclient/language/internal/rust/rust_test.go

```diff
@@ -910,6 +910,32 @@ Maybe they wanted to show some JSON:
 		"/// }",
 		"/// ```",
 	}
+
+	c := &Codec{}
+	got := c.FormatDocComments(input)
+	if diff := cmp.Diff(want, got); len(diff) > 0 {
```

Comment:
nit: this usually written this as `diff != ""`

Reply:
Fixed here and elsewhere in the PR. Will send a separate PR to clean up existing tests.